packages/cli/src/commands/generate/index.ts

```diff
@@ -15,14 +18,14 @@ export const generateSchema = async ({
     defaultSchema,
     schemas = [],
     tables = [],
+    schemaDefOverrides,
 }: GenerateSchemaOptions) => {
-    const result = await generate({
+    return await generate({
         defaultSchema,
         connectionString,
         includeSchemas: schemas,
         includeTables: tables,
-        outDir: out,
+        schemaDefOverrides: schemaDefOverrides,
+        outDir: path.resolve(path.join(process.cwd(), out)),
```

Comment:
jim you're not paying attention: this resolve/join/cwd pattern is buggy, it doesn't respect absolute paths.

Consider what happens when you use the absolute path "/tmp/foo.ts", it will be converted into ./tmp/foo.ts, which is incorrect.

I've mentioned this error in the past, please don't introduce this bug again.